RESTserver/src/main/java/com/softserve/if072/restservice/service/MessageService.java

```diff
@@ -0,0 +1,24 @@
+package com.softserve.if072.restservice.service;
+
+import com.softserve.if072.common.model.Message;
+import com.softserve.if072.common.model.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.messaging.simp.SimpMessagingTemplate;
+import org.springframework.stereotype.Service;
+
+import java.util.Date;
+
+@Service
+public class MessageService {
+    private SimpMessagingTemplate messaging;
+
+    @Autowired
+    public MessageService(SimpMessagingTemplate messaging) {
+        this.messaging = messaging;
+    }
+
+    public void broadcastSpittle(String msg, User user) {
```

Comment:
Could You please configure this Message Broadcasting in the MVC application instead of rest?
In the mvc application you will have the local of the user. You can get them from cookies, or you can store local from cookies to session, and use as session attribute.
Then add one more argument to this method local and use here MessageSource class from Spring, which can resolve codes into messages from different message_XXX.properties. Instead of msg as first argument, it should be message code to message.